README.md

```diff
@@ -1,2 +1,87 @@
-# dev-team
-Proposal for the Dev Team
+# Student Robotics Dev Team Proposal
+
+This document aims to solve the important issue of development by outlining the formation of a number of smaller sub-groups within Student Robotics:
+- The Development Coordination Team.
+- A number of _project groups_.
+
+## The Development Coordination team
+
+The Development Coordination team (a.k.a. the _Dev Team_) is a group of volunteers that are chosen annually in a similar fashion to the formation of the _Core Team_, although they may be responsible for projects beyond the annual timescale.
+
+The goals of the _Dev Team_ are to ensure the long-term longevity of Student Robotics by organising projects that enhance the experience for SR’s competitors and volunteers in line with its values, and enable SR’s long term growth. Additionally they will aim to fulfill the values of Student Robotics, in particular our goal of reflecting reality by incorporating the latest technology whenever feasible.
+
+## Project Groups
+
+A _project group_ is a smaller group of one or more self-selected volunteers who have a particular interest in working on something together. The _project group_ should report progress regularly to the _Dev Team_. The _Dev Team_ oversees the various _project groups_ to ensure that they are on track and aligned with the goals of the charity. _Project groups_ are responsible for maintaining projects that they have.
+
+### Formation of a Project Group
+
+A group of volunteers decides that they would like to work on a project. They communicate their ideas to the _Dev Team_ and the _Dev Team_ will decide whether the project is beneficial to the organisation. A schedule for the project will be discussed between the _Dev Team_ and the _project group_. A _project group_ may apply to the _Dev Team_ for budget for specific purchases for their project, given sufficient justification. The _Dev Team_ should also keep in mind the future needs of _project groups_ when planning their budget.
+
+Alternatively, the _Dev Team_ may identify the need for a project within the organisation and will find interested volunteers to form a group to do the project.
+
+## Budgeting
+
+The _Dev Team_ is allocated a budget, to be agreed with the Trustees. The scope of this budget is to cover costs incurred for development activities only, and not the cost of deploying any completed projects which would be arranged between the trustees and _Core Team_.
```

Comment:
```suggestion
The _Dev Team_ is allocated a budget, to be agreed with the Trustees. The scope of this budget is to cover costs incurred for development activities only, and not the cost of deploying any completed projects which would be arranged between the Trustees and _Core Team_.
```

Reply:
I disagree. One could argue that COBAL and python 2.3 are industry standard because they're probably used somewhere.